infra/base-images/base-builder/Dockerfile

```diff
@@ -29,6 +29,7 @@ RUN dpkg --add-architecture i386 && \
         jq \
         libc6-dev-i386 \
         patchelf \
+        python \
```

Comment:
This is not needed, latest python3 is installed already, see a few lines below.

Reply:
This change was meant to address the same problem discussed [here](https://github.com/google/oss-fuzz/pull/4936#discussion_r557816477) - basically Bazel requires `/usr/bin/python` to exist in the system.

Do you think we should just update the code below to create this as a symlink to `/usr/bin/python3`?

Reply:
So, add symlink before install and then remove it after install in the same RUN call using &&